mod/admin.php

```diff
@@ -1590,14 +1781,14 @@ function admin_page_logs(&$a){
 			}
 			fclose($fp);
 		}
-	}
+	}*/
 
```

Comment:
It would be cool if you could write a short comment why you disabled this piece of code. So we could understand it in the future why this was done and can decide what we want to do with it.